src/luascript.cpp

```diff
@@ -4248,25 +4250,18 @@ int LuaScriptInterface::luaGameCreateTile(lua_State* L)
 	return 1;
 }
 
-int LuaScriptInterface::luaGameStartRaid(lua_State* L)
```

Comment:
API breaking change. Should we provide a compat function?

Reply:
Maybe. There is `Game.startEvent` now, it could be an alias and expect people won't try `Game.startRaid("Server Save")`.